WcaOnRails/app/models/competition.rb

```diff
@@ -26,6 +26,22 @@ class Competition < ActiveRecord::Base
   validates :venue, format: { with: PATTERN_TEXT_WITH_LINKS_RE }
   validates :website, format: { with: /\Ahttps?:\/\/.*\z/ }, allow_blank: true
 
+  def self.NEARBY_DISTANCE_KM_WARNING
```

Comment:
@timhabermaas, I told @pedrosino to do this. Is there a better way of getting class constants that everyone can access?